### DISCUSSION TITLE
Detection and Visualization of CO2 Concentration Using Hyperspectral Satellite Data (Project 251)

### DISCUSSION THREAD

**Discussion by robertogl** [mathworks/MATLAB-Simulink-Challenge-Project-Hub#105] [Collaborate]
Contribute to the discussion by asking and/or answering questions, commenting, or sharing your ideas for solutions to [project 251](https://github.com/mathworks/MATLAB-Simulink-Challenge-Project-Hub/blob/main/projects/Detection%20and%20Visualization%20of%20CO2%20Concentration%20Using%20Hyperspectral%20Satellite%20Data)

---
*:robot: An AI bot may reply to your comments with suggestions and guidance. It will not solve the project for you. Your comments may be processed by a third-party LLM. To direct a question to a specific person, tag them with @username.*

**Comment by ishanporwal321** [comment]
Is there any compulsion of using MATLAB and it's toolbox in the project?

**Reply by robertogl** [reply]
Hi @ishanporwal321, 

It is not a 'compulsion' but a requirement if you want to receive our rewards. Thanks.

**Comment by lohithareddy2511** [comment]
Hi, I am Lohitha Reddy Karamala. My team and I worked together on this project, and I would appreciate it if you could take a look at it.

**Comment by Namrathay07** [comment]
Introduction to the Project
The Detection and Visualization of CO₂ Concentration Using Hyperspectral Satellite Data project focuses on processing satellite hyperspectral images to identify regions with high CO₂ concentrations and visualizing this information geospatially. By leveraging the Hyperspectral Imaging Library for Image Processing Toolbox™ and the Mapping Toolbox™, this project explores advanced techniques for CO₂ detection and quantification.
The goal is to develop a robust algorithm that applies various detection methods, including Cluster-Tuned Matched Filter (CTMF), Joint Reflectance and Gas Estimator (JRGE), Spectral Fitting Algorithm, and Continuum Interpolated Band Ratio (CIBR), to accurately estimate CO₂ levels from hyperspectral satellite data. The results will be mapped using MATLAB’s Mapping Toolbox™, providing a clear and intuitive visualization of CO₂ distribution across different regions.
